cmake/projects/NASM/schemes/url_sha1_nasm.cmake.in

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2017, Zhuhao Wang
```

Comment:
Actually this scheme might be not needed too. Please try this one:
* https://github.com/ruslo/hunter/blob/master/cmake/schemes/url_sha1_autotools.cmake.in